lib/resources/example_json/r4_examples_appointment.rb

```diff
@@ -1,5 +1,133 @@
 module Cerner
   module Resources
+    R4_APPOINTMENT_ENTRY ||= {
+   "resourceType":"Appointment",
+   "id":"20465903",
+   "meta":{
+      "versionId":"0",
+      "lastUpdated":"2018-03-02T21:51:55.000Z"
+   },
+   "text":{
+      "status":"generated",
+      "div":"&lt;div>&lt;p>&lt;b>Appointment&lt;/b>&lt;/p>&lt;p>&lt;b>Status&lt;/b>: booked&lt;/b>&lt;/p>&lt;p>&lt;b>Service Type&lt;/b>: Surgery&lt;/b>&lt;/p>&lt;p>&lt;b>Start&lt;/b>: Mar 27, 2018  9:00 A.M. CDT&lt;/b>&lt;/p>&lt;p>&lt;b>End&lt;/b>: Mar 27, 2018 10:00 A.M. CDT&lt;/b>&lt;/p>&lt;p>&lt;b>Participants&lt;/b>: OR Rooms, Adams, Charlotte, KM Main OR&lt;/p>&lt;/div>"
```

Comment:
update to reflect new narrative text

Reply:
updated participants https://github.com/cerner/fhir.cerner.com/pull/269/commits/6e094421299a9712e721c6d4a8d83ba8a4e36b87#diff-1a83ec9f4ba450cba6a16d8bcda1890cR12